.github/workflows/editoast.yml

```diff
@@ -46,7 +46,7 @@ jobs:
       - name: Setup rust toolchain
         uses: actions-rs/toolchain@v1
         with:
-          toolchain: stable
+          toolchain: nightly
```

Comment:
I think that we should let rust in stable version ? (unless I missed an episode)

Reply:
`set_alloc_error_hook()` is nightly only as explained in the doc : https://doc.rust-lang.org/std/alloc/fn.set_alloc_error_hook.html

I think that even for the `-Zoom=panic` it is still an unstable feature : https://doc.rust-lang.org/beta/src/alloc/alloc.rs.html#407

If editoast abort we can maybe see that there was an OOM by looking at the cluster statistics instead of forcing it to panic ? I agree that moving to `nightly` does not look great :/